src/main/java/card/validation/ValidateCard.java

```diff
@@ -0,0 +1,103 @@
+package main.java.card.validation;
+
+import main.java.card.issuer.CardIssuerParameters;
+import main.java.card.information.CreditCardInformation;
+import main.java.card.issuer.CardIssuers;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class ValidateCard {
+
+    public enum ResultTypeEnum {
+        SUCCESS("Validated"),
+        ERR_ISUER("Incorrect issuer name"),
+        ERR_LENGTH("Incorrect card length"),
+        ERR_LUHN("Card number didn't successfully pass Luhn validation");
+
+        ResultTypeEnum(String msg){
+            message = msg;
+        }
+        public String getMessage() {
+            return message;
+        }
+
+        private String message;
+    }
+
+
+    public static ResultTypeEnum performValidation(String[] inputArgs){
+        CardInputArgumentsControl inputControl = new CardInputArgumentsControl(inputArgs, 2);
+
+        try {
+            inputControl.checkInput();
+        }
+        catch(IOException e){
+            System.out.println(e);
+            System.exit(1);
+        }
+
+        CreditCardInformation creditCard = new CreditCardInformation(inputArgs);
+
+        if(!CardIssuers.doesIssuerExist(creditCard.getCardIssuerName()))
+            return ResultTypeEnum.ERR_ISUER;
+
+        if (!checkCardNumberLength(creditCard))
+            return ResultTypeEnum.ERR_LENGTH;
+
+        if (!validateCardNumber(creditCard.getCardNumber()))
+            return ResultTypeEnum.ERR_LUHN;
+
+        return ResultTypeEnum.SUCCESS;
+    }
+
+    private static boolean checkCardNumberLength(CreditCardInformation creditCard){
```

Comment:
Rename method to `isValidCardNumberLength()`. 
Local variables _currentCardNumberLength_ and _currentCardIssuerName_ are unnecessary
Consider simplify method ie:
```suggestion
    private static boolean isValidCardNumberLength(CreditCard creditCard){
        List<Integer> availableCardNumberLengths = CardIssuers.getIssuerParameters(creditCard.getIssuerName()).getLengthOfCardNumber();
        return availableCardNumberLengths.contains(creditCard.getNumber().length());
    }
```